packages/editor/src/client/atoms/searchAndFilter/articleListFilter.tsx

```diff
@@ -0,0 +1,137 @@
+import CloseIcon from '@rsuite/icons/legacy/Close'
```

Comment:
Are the article, page, and peer article filters all the same? Seems like it would make sense to have a generic filter and avoid repeating it, no?

Reply:
yes. I totally agree. We could think of a component where you can pass an array with all the filter you wanna display.